### PR TITLE
Add project template for Prism.Autofac.Forms

### PR DESCRIPTION
**Pull request by nuitsjp**
As title.
This is for "Need project template other than Unity #890 ".

### CODE REVIEW

File: Extensibility/TemplatePack/ProjectTemplates/CSharp/Forms/PrismAutofacApp/AutofacApp/App.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        protected override void OnInitialized()
+        {
+            InitializeComponent();
+
+            NavigationService.NavigateAsync("MainPage?title=Hello%20from%20Xamarin.Forms");
+        }
+
+        protected override void RegisterTypes()
+        {
+            var builder = new ContainerBuilder();
```
Comment (nuitsjp) [inline Extensibility/TemplatePack/ProjectTemplates/CSharp/Forms/PrismAutofacApp/AutofacApp/App.xaml.cs]: In the case of the current Autofac, an error occurs unless you register ViewModel.
Is this something wrong with me?
If I am correct, I should fix AutofacExtensions.
It becomes a problem when registering View with ItemTemplate.

**Comment by nuitsjp** [comment]
In the case of the current Autofac, an error occurs unless you register ViewModel.
Is this something wrong with me?

```cs
        protected override void RegisterTypes()
        {
            var builder = new ContainerBuilder();
            builder.RegisterType<MainPageViewModel>();
            builder.Update(Container);

            Container.RegisterTypeForNavigation<MainPage>();
        }

```

If I am correct, I should fix AutofacExtensions.
It becomes a problem when registering View with ItemTemplate.

**Comment by bartlannoeye** [comment]
@nuitsjp the issue you're mentioning is fixed and will be in the next (pre)release. See https://github.com/PrismLibrary/Prism/issues/875

**Comment by nuitsjp** [comment]
Oh. I understand. Thank you.
Okay. Based on that premise, I will also modify the template.

**Comment by nuitsjp** [comment]
Remove ViewModel container registration from App.xaml.cs
Please, Review!

**Comment by brianlagunas** [comment]
Thanks for the PR.  I will review as soon as I can.

**Comment by brianlagunas** [comment]
Please make sure all .vstemplate files have unique template id's. If they are not unique, the templates will not work.

**Comment by nuitsjp** [comment]
I'm sorry.
Delete it once and update again.

**Pull request closed without merging**